test/integration/mssql-ha/controls/mssql.rb

```diff
@@ -12,46 +12,118 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require 'json'
+
 project_id = attribute('project_id')
 basename   = attribute('name')
 authorized_network = attribute('authorized_network')
+region = "us-central1"
+
+activation_policy = "ALWAYS"
+availability_type = "REGIONAL"
+data_disk_size_gb = 10
+data_disk_type = "PD_SSD"
+kind = "sql#settings"
+pricing_plan = "PER_USE"
```

Comment:
I don't think we need to test settings we're not really modifying.

Reply:
You're right but there are few place where a subset of these are modified. I can do another PR to clean this up.